### PR TITLE
chore: remove remaining instances of XULElem

### LINKED ISSUE #8820: chore: remove remaining instances of XULElem
The `XULElem` macro is still used in the following places and should be removed as it is deprecated:

| macro | yari | en-us | es | fr | ja | ko | pt-br | ru | zh-cn | zh-tw |
|:----- | ----:| -----:| --:| --:| --:| --:| -----:| --:| -----:| -----:|
| XULElem 🗑 |  |  | <span title="/web/css/-moz-user-focus/index.md …">1</span> | <span title="/mozilla/firefox/releases/7/index.md …">3</span> | <span title="/mozilla/firefox/releases/4/the_add-on_bar/index.md …">4</span> | <span title="/web/api/html_drag_and_drop_api/drag_operations/index.md …">1</span> |  |  |  |  |

### PR DESCRIPTION
**Pull request by schalkneethling**
The XULElem macro is still used in the following places and should be removed as it is deprecated.

fix #8820

### CODE REVIEW

File: files/fr/mozilla/firefox/releases/7/index.md [hunk 1]
```diff
@@ -107,8 +107,8 @@ Ces changements affectent les développeurs d'extensions ainsi que les développ
 
 ### XUL
 
-- Les éléments {{xulelem("tree")}} peuvent désormais conserver l'état des triangles d'affichage si les nœuds référencés par les {{XULAttr("datasources")}} ont tous des ID uniques spécifiés par les attributs "id".
-- Les éléments {{xulelem("panel")}} peuvent désormais être configurés pour [permettre à l'utilisateur de les faire glisser en cliquant n'importe où sur leur arrière-plan](/fr/docs/XUL/PopupGuide/Panels#Letting_panels_be_dragged_by_grabbing_the_background) avec le nouvel attribut {{XULAttr("backdrag")}}.
+- Les éléments `tree` peuvent désormais conserver l'état des triangles d'affichage si les nœuds référencés par les {{XULAttr("datasources")}} ont tous des ID uniques spécifiés par les attributs "id".
```
Comment (cw118) [inline files/fr/mozilla/firefox/releases/7/index.md:110]:
```suggestion
- Les éléments `<tree>` peuvent désormais conserver l'état des triangles d'affichage si les nœuds référencés par les {{XULAttr("datasources")}} ont tous des ID uniques spécifiés par les attributs "id".
```

File: files/fr/mozilla/firefox/releases/7/index.md [hunk 2]
```diff
@@ -107,8 +107,8 @@ Ces changements affectent les développeurs d'extensions ainsi que les développ
...
+- Les éléments `panel` peuvent désormais être configurés pour [permettre à l'utilisateur de les faire glisser en cliquant n'importe où sur leur arrière-plan](/fr/docs/XUL/PopupGuide/Panels#Letting_panels_be_dragged_by_grabbing_the_background) avec le nouvel attribut {{XULAttr("backdrag")}}.
```
Comment (cw118) [inline files/fr/mozilla/firefox/releases/7/index.md:111]:
```suggestion
- Les éléments `<panel>` peuvent désormais être configurés pour [permettre à l'utilisateur de les faire glisser en cliquant n'importe où sur leur arrière-plan](/fr/docs/XUL/PopupGuide/Panels#Letting_panels_be_dragged_by_grabbing_the_background) avec le nouvel attribut {{XULAttr("backdrag")}}.
```

File: files/fr/web/css/-moz-user-focus/index.md [hunk 3]
```diff
@@ -26,8 +26,6 @@ La propriété **`-moz-user-focus`** est utilisée pour indiquer si l'élément
 
 En utilisant la valeur `ignore`, on peut désactiver la prise de focus sur l'élément (l'utilisateur ne pourra pas activer l'élément) et l'élément sera sauté lors de la navigation à la tabulation.
 
-> **Note :** Cette propriété ne fonctionne pas pour les éléments XUL {{XULElem("textbox")}} car l'élément `textbox` en tant que tel ne reçoit jamais le focus. À la place, XBL crée un élément HTML {{HTMLElement("input")}} anonyme à l'intérieur du `textbox` et que l'élément reçoit le focus. On peut empêcher le `textbox` de prendre le focus clavier en passant son index de tabulation à `-1`, pour l'empêcher de prendre le focus souris, on pourra utiliser les événements `mousedown`.
-
 ## Syntaxe
```
Comment (cw118) [inline files/fr/web/css/-moz-user-focus/index.md:29]:
```suggestion
{{CSSInfo}}

## Syntaxe
```

**Review by cw118 — APPROVED** [review]
Thanks so much @schalkneethling ! I suggested three minor (not blocking) changes for `l10n-fr` :)

**Review by mfuji09 — APPROVED** [review]
LGTM for `ja` files

**Review by hochan222 — APPROVED** [review]
LGTM for ko file

**Review by Graywolf9 — APPROVED** [review]
LGTM for `es` Thank you!

**Comment by yin1999** [comment]
Approved by all those locales, I've resolved some conflicts for `fr` and `ja` (one per locale). poke @cw118 @mfuji09 for the last check.

**Review by cw118 — APPROVED** [review]
LGTM for fr :)

**Comment by schalkneethling** [comment]
Thank you for the reviews everyone!

**Pull request merged**